src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -29,9 +33,21 @@ public class FunctionsSyncManager : IFunctionsSyncManager, IDisposable
         private const string HubName = "HubName";
         private const string TaskHubName = "taskHubName";
         private const string Connection = "connection";
-        private const string DurableTaskStorageConnectionName = "azureStorageConnectionStringName";
+        private const string DurableTaskV1StorageConnectionName = "azureStorageConnectionStringName";
+        private const string DurableTaskV2StorageOptions = "storageOptions";
+        private const string DurableTaskV2StorageConnectionName = "connectionStringName";
         private const string DurableTask = "durableTask";
 
+        private readonly Regex versionRegex = new Regex(@"Version=(?<majorversion>\d)\.\d\.\d");
+
+        internal const string InvalidDurableV1ConfigErrorMsg = "Invalid host.json configuration for 'durableTask' for durable task extension 1.x";
```

Comment:
Just have a single format string that takes the version as a param

Reply:
I decided that throwing an exception when the config is invalid for the version is outside the scope of this API, as that is already handled appropriately by the extension. 

Instead, I more safely look at the host.json so that we shouldn't get parsing exceptions.